LuaSkin/LuaSkin/Skin.m

```diff
@@ -294,6 +338,15 @@ - (void)checkArgs:(int)firstArg, ... {
                 break;
             case LUA_TNUMBER:
                 lsType = LS_TNUMBER;
+
+                if ((spec & lsType) && (spec & LS_TINTEGER)) {
+                    int isInteger ;
+                    lua_tointegerx(_L, idx, &isInteger) ;
+                    if (!isInteger) {
```

Comment:
So I'm thinking that `static int module_foo()` calls `checkArgs` with `LS_TNUMBER | LS_TINTEGER` and from Lua, someone calls `hs.module.foo(9.1)`. I think this change will see that `LS_TINTEGER` is specified, look at the `9.1` and set `isInteger` to false, which causes us to raise a `luaL_error()`. I haven't tested whether this is true, it was just a thought that occurred to me as I was reading the PR.